docs/Installation Guide.md

```diff
@@ -99,6 +99,10 @@ You may come across a message that says `default: Warning: Remote connection dis
 
 File syncing will work properly after you receive this message: `default: Mounting shared folders`. Please wait for this message before proceeding to the next steps.
 
+The virtual machine will ask you for login id and password in case you want to work with the virtual machine.
```

Comment:
the who para might not be needed i think. 
Simply mentioning


"default username:vagrant
default password:vagrant"

concise is better in my opinion